magner/lib/types/configs/pages/shared.ts

```diff
@@ -1,5 +1,6 @@
 import type { TranslateData } from 'lib/utils/core/translate';
 import type { RouteLocationRaw, RouteLocation } from 'vue-router';
+import { ActionButton } from 'lib/types/utils/actions';
```

Comment:
Todo: add `type` keyword for import. It is needed everywhere. Sometimes the build fails because of this keyword